src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+type Props = {
+  todo: User;
```

Comment:
It's user, not todo:
```suggestion
  user: User;
```